packages/binding-opcua/src/opcua-protocol-client.ts

```diff
@@ -63,20 +63,11 @@ export type NodeIdLike2 = NodeIdLike & {
 };
 
 export interface FormPartialNodeDescription {
-    "opcua:nodeId": NodeIdLike | NodeByBrowsePath;
+    "opcua:nodeId"?: NodeIdLike | NodeByBrowsePath; // TODO remove in future, kept for backwards compatibility only
```

Comment:
I think this is a perfect use of https://www.typescriptlang.org/play/#example/jsdoc-deprecated

Reply:
Done in https://github.com/eclipse-thingweb/node-wot/pull/1368/commits/2470c80a90b5eb760bda2f32517be95376fc5fcb